### PR TITLE
Documentation for the new WebDAV camera.

### PR DESCRIPTION
**Pull request by jkeljo**
**Description:**
Documentation for the new WebDAV camera.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#22607

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
Parent PR still open

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_components/camera.webdav.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+description: "Instructions on how to pull images from a WebDAV share within Home Assistant."
+date: 2019-03-31 20:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: webdav.jpg
+ha_category: Camera
+ha_iot_class: Local Polling
+ha_release: 0.92
```
Comment (klaasnicolaas) [inline source/_components/camera.webdav.markdown:13]:
```suggestion
ha_release: 0.96
```

**Comment by frenck** [comment]
Closing PR, since the parent PR has been closed.

**Pull request closed without merging**